README.md

```diff
@@ -237,9 +214,8 @@ jobs:
         with:
           results_file: results.sarif
           results_format: sarif
-          # (Optional) "write" PAT token. Uncomment the `repo_token` line below if:
+          # (Optional) fine-grained personal access token. Uncomment the `repo_token` line below if:
           # - you want to enable the Branch-Protection check on a *public* repository, or
-          # - you are installing Scorecards on a *private* repository
           # To create the PAT, follow the steps in https://github.com/ossf/scorecard-action#authentication-with-pat.
```

Comment:
this link will need changed as you've changed the section header above

Reply:
Can you do a search for the link actually, it shows up in multiple places. (and is currently broken)

Reply:
Done.